### PR TITLE
Fluentd plugin for http-out

### PR DESCRIPTION
**Pull request by cjohannsen81**
Attempt to add ruby gem for fluentd to enable http out traffic for /metrics

#### For version bump PRs

- [X] The `epoch` field is reset to 0

### CODE REVIEW

File: fluent-plugin-out-http.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    with:
+      gem: ${{package.name}}
+      version: ${{package.version}}
+
+  - uses: ruby/clean
+
+update:
+  enabled: true
+  github:
+    identifier: splunk/fluent-plugin-out-http
```
Comment (found-it) [inline fluent-plugin-out-http.yaml:52]: I think this should be `fluent-plugins-nersery/fluent-plugin-out-http` so the auto-updater can locate the repo on github

**Review by found-it — DISMISSED** [review]

**Review by cjohannsen81 — COMMENTED** [review]
You are right. Pipeline failed because of that.

File: fluent-plugin-out-http.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    with:
+      gem: ${{package.name}}
+      version: ${{package.version}}
+
+  - uses: ruby/clean
+
+update:
+  enabled: true
+  github:
+    identifier: fluent-plugins-nursery/fluent-plugin-out-http
```
Comment (found-it) [inline fluent-plugin-out-http.yaml:52]: Looks like it's defaulting to releases which the upstream repo doesn't have. Probably need to add this flag

```suggestion
    identifier: fluent-plugins-nursery/fluent-plugin-out-http
    use-tag: true
```

**Review by found-it — CHANGES_REQUESTED** [review]

File: fluent-plugin-out-http.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      - ruby-3.2-dev
+      - build-base
+      - busybox
+      - git
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 821f4468c2db1ddcfaad8520b1d9a5b33f5d6eda89a2fc931da735b3d42a6f44
+      uri: https://github.com/fluent-plugins-nursery/fluent-plugin-out-http/archive/refs/tags/${{package.version}}.tar.gz
```
Comment (found-it) [inline fluent-plugin-out-http.yaml:34]:
```suggestion
      uri: https://github.com/fluent-plugins-nursery/fluent-plugin-out-http/archive/refs/tags/${{package.version}}.tar.gz
      uri: https://github.com/fluent-plugins-nursery/fluent-plugin-out-http/archive/refs/tags/v${{package.version}}.tar.gz
```

Comment (found-it) [reply fluent-plugin-out-http.yaml:34]: I _think_ you need a `v` in front of the version here

Comment (found-it) [reply fluent-plugin-out-http.yaml:34]: That looks to be why the build is failing

**Review by found-it — APPROVED** [review]

**Pull request merged**